### PR TITLE
Cc2024

### PR DESCRIPTION
**Pull request by teresa-m**
update of the BioNT logo and typos in the news for CC2024

### CODE REVIEW

**Comment by teresa-m** [comment]
@hexylena or @shiltemann: I would like to have a 'small' heading in the middle of this post. However, only the first level heading does not give an error/warning. If you have any suggestions, on how to fix this it would be great. But you can also leave it as it is. 
@shiltemann thanks for spotting the typos.

File: news/_posts/2024-07-11-CC2024-registration.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ contributions:
...
 ---
 
-Are you interested in training and want to connect with other enthusiastic traineres? Than join the BioNT Community Event & CarpentryConnect - Heidelberg 2024. The Carpentries and other learning communities will meet to network and collaborate during this 3-day event. The GTN will is presten with a Poster, Lightning Talks and a Mini-hackathons.
+Are you interested in training and want to connect with other enthusiastic trainers? Then join the BioNT Community Event & CarpentryConnect - Heidelberg 2024. The Carpentries and other learning communities will meet to network and collaborate during this 3-day event. The GTN will be present with a Poster, Lightning Talks and a Mini-hackathon.
 
 [Check out the event](https://biont-training.eu/CarpentryConnect2024.html){:.btn.btn-info}
 You can register to participate remotely or in-person before 15th September.
 
-### If you would like to learn more about the event please also see the previous Carpentries posts on this topic:
+# If you would like to learn more about the event please also see the previous Carpentries posts on this topic:
```
Comment (shiltemann) [inline news/_posts/2024-07-11-CC2024-registration.md:14]:
```suggestion
#### If you would like to learn more about the event please also see the previous Carpentries posts on this topic:
```

**Comment by shiltemann** [comment]
Hi @teresa-m, thanks! 

We can just ignore the error about the heading level, that is more important for tutorials than here, so don't worry about it too much

**Review by shiltemann — APPROVED** [review]

**Comment by teresa-m** [comment]
Thanks, good to know! Thanks a lot!

**Pull request merged**